src/nidcpower/templates/session.py/fancy_fetch.py.mako

```diff
@@ -19,6 +19,9 @@
     else:
         raise ValueError('Only fetch_multiple and measure_multiple are supported. Got {0}'.format(f['python_name']))
 %>\
+% if f['use_session_lock']:
```

Comment:
I dislike the fact that we use two different terms for the same thing.
Sometimes we call it "session lock" and sometimes "IVI lock". I don't think it's going to be obvious to all contributors that these refer to the same thing.

Probably not worth changing right now, and I don't have a better name for the decorator.